docs/synapse/userguides/storm_ref_lift.ipynb

```diff
@@ -513,6 +510,129 @@
     "podes = await core.eval(q, num=3, cmdr=False)"
    ]
   },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "*Lift all nodes with a specific tag property:*\n",
+    "\n",
+    "- Lift all nodes with any tag that has a ``:risk`` tag property:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "hideCode": true
+   },
+   "outputs": [],
+   "source": [
+    "# Create a custom tag property\n",
+    "await core.core.addTagProp('risk', ('int', {'minval': 0, 'maxval': 100}), {'doc': 'Risk score'})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "hideCode": true,
+    "hideOutput": true
+   },
+   "outputs": [],
+   "source": [
+    "# Make some nodes\n",
+    "q = '[ ( inet:fqdn=aoldaily.com +#rep.symantec:risk = 88 ) ( inet:fqdn=google.com +#rep.domaintools:risk=4 ) ]'\n",
+    "# Run the query and test\n",
+    "podes = await core.eval(q, num=2, cmdr=False)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "hideCode": true
+   },
+   "outputs": [],
+   "source": [
+    "# Define and print test query\n",
+    "q = '#:risk'\n",
+    "print(q)\n",
+    "# Execute the query and test\n",
+    "podes = await core.eval(q, num=2, cmdr=False)"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "*Lift all nodes with a specific tag property and value(s):*\n",
+    "\n",
+    "- Lift all nodes with a tag that has a ``:risk`` property with a value greater than or equal to 80:"
```

Comment:
Should probably add "tag" in between "``:risk``" and "property" to match up with the rest of the examples.